src/circuit/gadget/sinsemilla/chip/generator_table.rs

```diff
@@ -1,16 +1,11 @@
-use crate::primitives::sinsemilla::{self, sinsemilla_s_generators, S_PERSONALIZATION};
```

Comment:
`sinsemilla_s_generators` is now only used in tests; remove it from the public API.